analyzer/optane/check_dimm_MediaTemperature_Injection_Counter

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+#set -x
+
+# Prevent repeated sourcing
+if [ -n "${_MODULE_CHECK_DIMM_MEDIA_TEMP_INJECTION_COUNTER_}" ]; then
+  return
+fi
+_MODULE_CHECK_DIMM_MEDIA_TEMP_INJECTION_COUNTER_="Loaded"
+
+# Validate Temp Injection Counter != 0
+function optane_check_dimm_media_temperature_injections_count() {
+  local FNAME=$1        # File name to process
+  local ERR_STATE=false # Used for error reporting
+
+  # Exit the function with INFO message if the file to process doesn't exist
+  if [[ ! -f ${FNAME} ]] ; then
+    info_msg " Could not process '${FNAME}'. File not found"
+    REPORT_COUNT_INFO=$((REPORT_COUNT_INFO+1)) # Increment the INFO count
+    return
+  fi
+
+  # Create Associative Array
+  declare -A MEDIA_TEMP_INJ_COUNETER  # Key is the DIMM Property value; Value is a list of DIMMs with this Property value
+
+  # Get the DimmID, Property, and Value fields from the input file
+  while IFS='|' read -r DimmID Property Value 
+  do
+    DimmID="$(echo ${DimmID})"         # Remove leading and trailing whitespaces
+    Property="$(echo ${Property})" # Remove leading and trailing whitespaces
+    PropertyValue="$(echo ${Value})"   # Remove leading and trailing whitespaces
+    if [ ${OPT_VERBOSITY} -ge 3 ]; then
+      debug_msg "DimmID ${DimmID}: ${Property}=${Value}"
+    fi
+    # Report PMem Modules that have PackageSparingCapable  = 0 (False/Disabled) 
```

Comment:
Update the comment to reflect MediaTemperatureInjectionsCounter vs PackageSparingCapable